distributions/Dist_kas938.py

```diff
@@ -1,23 +1,21 @@
 import numpy as np
 from .base_distribution import BaseDistribution
 
-class Dist_kc90(BaseDistribution):
+class Dist_kas938(BaseDistribution):
 	def __init__(self):
-		self.f_max = 1
-		self.x_min = -1
-		self.x_max = 1
+		self.f_max = 1.019
+		self.x_min = 0
+		self.x_max = 3
 
 
 	def pdf(self, x):
 		"""This is your PDF"""
-		return np.abs(x)
+		return ( math.exp(-x**2) + math.exp(-2*(x-2)**2) )
```

Comment:
you would need np.exp here instead of math.exp for the broadcasting to work. 
is this normalized?